packages/apps/patterns/react-appkit/src/telemetry/telemetry.ts

```diff
@@ -11,9 +12,13 @@ export const BASE_TELEMETRY_PROPERTIES: any = {};
 
 if (navigator.storage?.estimate) {
   setInterval(async () => {
-    const storageEstimate = await navigator.storage.estimate();
-    BASE_TELEMETRY_PROPERTIES.storageUsage = storageEstimate.usage;
-    BASE_TELEMETRY_PROPERTIES.storageQuota = storageEstimate.quota;
+    try {
+      const storageEstimate = await navigator.storage.estimate();
+      BASE_TELEMETRY_PROPERTIES.storageUsage = storageEstimate.usage;
+      BASE_TELEMETRY_PROPERTIES.storageQuota = storageEstimate.quota;
+    } catch (error) {
+      log.warn('Failed to run estimate()', error);
```

Comment:
Note that sentry hooks into error/warn logs so this is probably going to generate a bunch of sentry stuff for the extension. Might want to just make it a debug log with a comment about when we expect it to fail